src/client/features/search/index.js

```diff
@@ -104,21 +104,26 @@ class Search extends React.Component {
     }, props.search);
 
     const searchResults = state.searchResults.map(result => {
-      const dsInfo = _.find(state.dataSources, ds => {
-        return ds.uri === result.dataSource[0];
-      });
-      return h('div.search-item', [
-        h('div.search-item-icon',[
-          h('img', {src: dsInfo.iconUrl})
-        ]),
-        h('div.search-item-content', [
-          h(Link, { to: { pathname: '/view', search: queryString.stringify({ uri: result.uri }) }, target: '_blank' }, [
-            h('h3.search-item-content-title', result.name || 'N/A'),
+      if(_.isEmpty(state.dataSources)){
+        return h('div');
```

Comment:
might make sense to keep returning relevant info such as title, etc, but show default datasource data if it couldnt be loaded (blank image, omit datasource name in the search hit)